### PR TITLE
DP-27930: A11y - Focusable hidden elements in feedback module

### PR DESCRIPTION
**Pull request by arthurbaghdas**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test

1.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/assets/scss/03-organisms/_mass-feedback-form.scss [hunk 1]
```diff
@@ -58,10 +58,11 @@ $formQuickTransitions: max-height 0.3s linear;
 
 
     max-height: 0;
-    overflow: hidden;
+    display: none;
```
Comment (ygannett) [inline packages/assets/scss/03-organisms/_mass-feedback-form.scss:59]: @arthurbaghdas As you add `display: none;`, you can remove L. 60 `max-height: 0;` the line is corresponding to `overflow: hidden;` .

File: packages/assets/scss/03-organisms/_mass-feedback-form.scss [hunk 2]
```diff
@@ -58,10 +58,11 @@ $formQuickTransitions: max-height 0.3s linear;
...
     transition: $formTransitions;
 
     &.is-open {
+      display: initial;
       max-height: 150vh;
```
Comment (ygannett) [inline packages/assets/scss/03-organisms/_mass-feedback-form.scss:66]: As you add `display: none;` to `div.ma__mass-feedback-form__fields`, it doesn't make sense to put `display: initial;`. It's better to be more expressive to have `dipslay: block;` here.  

I think L. 66 `max-height: 150vh;` is responding to `max-height: 0;` and `overflow: hidden;` for `div.ma__mass-feedback-form__fields`.  This line can be removed.

**Review by ygannett — CHANGES_REQUESTED** [review]
Please see my comments.

File: packages/assets/scss/03-organisms/_mass-feedback-form.scss (hunk 2 above)
Comment (ygannett) [inline packages/assets/scss/03-organisms/_mass-feedback-form.scss]: I think L. 66 `max-height: 150vh;` is responding to `max-height: 0;` and `overflow: hidden;` for `div.ma__mass-feedback-form__fields`. This line can be removed.

File: packages/assets/scss/03-organisms/_mass-feedback-form.scss (hunk 1 above)
Comment (arthurbaghdas) [reply packages/assets/scss/03-organisms/_mass-feedback-form.scss:59]: Ok

File: packages/assets/scss/03-organisms/_mass-feedback-form.scss (hunk 2 above)
Comment (arthurbaghdas) [reply packages/assets/scss/03-organisms/_mass-feedback-form.scss]: Removed

**Review by ygannett — APPROVED** [review]
@arthurbaghdas Thank you for the changes. 
LGTM.

**Pull request merged**